pkg/services/rocketchat.go

```diff
@@ -0,0 +1,134 @@
+package services
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/RocketChat/Rocket.Chat.Go.SDK/models"
```

Comment:
Would you sort the order as same other package, please?
```
import (
	standard library
	external library
	internal library
)
```

Reply:
Fixed